docs/src/index.md

```diff
@@ -0,0 +1,118 @@
+- [`FK5Coords`](@ref): FK5 coordinates system (with arbitrary equninox)
+
+Each type holds a longitude and latitude, and each is a subtype of
+[`AbstractSkyCoords`](@ref).
+
+```jldoctest
+julia> c1 = ICRSCoords(0., 0.)  # inputs are ra, dec in radians
+ICRSCoords{Float64}(0.0, 0.0)
+
+julia> c1.ra # access ra, dec individually
+0.0
+
+julia> c2 = convert(GalCoords, c1) # convert to a different system
+GalCoords{Float64}(1.6814027872278692, -1.0504884034813007)
+
+julia> c2.l # Note that galactic coordinate fields are l, b
+1.6814027872278692
+
+julia> convert(FK5Coords{2000}, c1)
+FK5Coords{2000,Float64}(1.1102233723050067e-7, 4.411803426976326e-8)
+
+```
+
+### Angular Separation between Coordinates
+
+The [`separation`](@ref) function allows you to compute the angular (great-circle)
+distance between two coordinates, in radians, using
+the [Vincenty formula](http://en.wikipedia.org/wiki/Great-circle_distance).  The
+coordinates can be also given in different systems.  For example, according to
+SIMBAD the `FK5Coords{2000}` coordinates
+of [Mizar](http://simbad.u-strasbg.fr/simbad/sim-id?Ident=MIZAR) are
+
+```jldoctest sep
+julia> mizar = FK5Coords{2000}(3.507787, 0.958628)
+FK5Coords{2000,Float64}(3.507787, 0.958628)
+
+```
+
+while the `GalCoords` coordinates of Alcor are
+
+```jldoctest sep
+julia> alcor = GalCoords(1.968189, 1.072829)
+GalCoords{Float64}(1.968189, 1.072829)
+
+```
+
+Their angular separation is given by
+
+```jldoctest sep
+julia> separation(mizar, alcor) # Radians
+0.0034353091694529292
+
+julia> rad2deg(separation(mizar, alcor)) * 60 # Arcminutes
+11.8097230039347
+```
+
+## Accuracy
+
+All the supported conversions have been compared to the results of
+[astropy.coordinates](https://docs.astropy.org/en/stable/coordinates/) (to better than 0.0001 arcsec agreement for `Float64`).
+In turn, [astropy.coordinates](https://docs.astropy.org/en/stable/coordinates/) has been tested against many other tools.
+
+## Performance
+
+For small and moderate numbers of coordinates, conversions are much
+faster than [astropy.coordinates](https://docs.astropy.org/en/stable/coordinates/) in Python. The following plot shows the
+performance for converting ICRS coordinates to various other systems
+(Galactic, FK5J2000 and FK5J1975), using [astropy.coordinates](https://docs.astropy.org/en/stable/coordinates/) (`py_*`
+labels) and SkyCoords.jl (`jl_*` labels). The x axis denotes the
+number of coordinates being simultaneously converted, with 1
+corresponding to scalar coordinates.
+
+![times](assets/bench.png)
+
+| Specs           |                                        |
+|:----------------|:---------------------------------------|
+| CPU             | Intel core i5-8259U @ 2.3GHz (4 cores) |
+| RAM             | 16GB                                   |
+| Julia Version   | 1.2 (JULIA\_NUM\_THREADS=8)            |
+| Python Version  | 3.7                                    |
+| Astropy Version | 3.1.2                                  |
```

Comment:
I like this table!  However, I'd remove the reference to `JULIA_NUM_THREADS` as it may lead people to think that we're using multiple threads here, but this isn't the case: #12